README.md

```diff
@@ -3,6 +3,8 @@
 [![CircleCI](https://circleci.com/gh/solidusio-contrib/solidus_paypal_marketplace.svg?style=shield)](https://circleci.com/gh/solidusio-contrib/solidus_paypal_marketplace)
 [![codecov](https://codecov.io/gh/solidusio-contrib/solidus_paypal_marketplace/branch/main/graph/badge.svg?token=ywAGAmYt9W)](https://codecov.io/gh/solidusio-contrib/solidus_paypal_marketplace)
 
+**WARNING**: This extension has not been released onto RubyGems and is not currently being developed upon.
```

Comment:
Can you also update this [line](https://github.com/solidusio-contrib/solidus_paypal_marketplace/pull/105/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R23) with the command to use the github URL?

Reply:
I don't see what line you are referring to @DanielePalombo

Reply:
Sorry, the line is:

https://github.com/solidusio-contrib/solidus_paypal_marketplace/blob/5fe7f23cbbed02b324a93beebaaaa84b13867ce8/README.md?plain=1#L23